src/rasenmaeher_api/db/enrollments.py

```diff
@@ -37,11 +38,45 @@ async def create_enrollment(self, callsign: str) -> "Enrollment":
             raise Deleted("Can't create enrollments on deleted pools")
         return await Enrollment.create_for_callsign(callsign, self.pk, self.extra)
 
-    async def set_active(self, state: bool = True) -> Self:
+    async def set_active(self, state: bool) -> Self:
         """Set active and return refreshed object"""
         await self.update(active=state).apply()
         return await EnrollmentPool.by_pk(self.pk, allow_deleted=True)
 
+    @classmethod
+    async def by_owner(cls, owner_uuid: str) -> Self:
+        """Get by owner UUID"""
+        obj = await EnrollmentPool.query.where(EnrollmentPool.owner == owner_uuid).gino.first()
+        if not obj:
+            raise NotFound()
+        return cast(EnrollmentPool, obj)
+
+    @classmethod
+    async def by_invitecode(cls, invitecode: str) -> Self:
+        """Get by invitecode"""
+        obj = await EnrollmentPool.query.where(EnrollmentPool.invitecode == invitecode).gino.first()
+        if not obj:
+            raise NotFound()
+        return cast(EnrollmentPool, obj)
+
+    @classmethod
+    async def create_or_update_invitecode_for_callsign(
+        cls, callsign: str, extra: Optional[Dict[str, Any]] = None
+    ) -> Self:
+        """Create enrollment pool owned by callsign, or update invitecode for existing one"""
+        _owner = await Person.by_callsign(callsign=callsign)
+
+        _invitecode = "".join(secrets.choice(CODE_ALPHABET) for _ in range(CODE_CHAR_COUNT))
```

Comment:
only start variable with underscore if it's not meant to be used right now but will be Real Soon Now. If it's not meant to be used at all in general either assign it to just "_" (if it has to be assigned, like fixtures in unit tests) or don't assign it at all.

Reply:
Got it. Did a bit of renaming here and there.